webapp/src/types/plugin/service.d.ts

```diff
@@ -0,0 +1,16 @@
+type PluginApiServiceName = 'getMe';
+
+type ApiErrorResponse = {
+    data: {
+        error: string,
+    },
+    status: number,
+}
+
+type CustomErrorModules = 'getMe';
```

Comment:
are we using this ? If not please remove this and if in future we may need it then write a TODO comment here .
All the review comments are for the whole project not just for that particular line. So check this everywhere and fix it